src/test/java/core/basesyntax/service/strategy/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.strategy.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.strategy.OperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static final Map<Fruit, Integer> expectedMap = new HashMap<>();
+    private final OperationHandler balanceOperationHandler = new BalanceOperationHandler();
+
+    @Test
+    public void doOperation_validData_ok() {
+        String fruit = "apple";
+        String quantity = "75";
```

Comment:
i think you can use constant here, like private static final String QUANTITY = 75; and than whan you need quantity lower than zero, you can use the same , just like that : balanceOperationHandler.doOperation(fruit, - QUANTITY);